src/ft2font.cpp

```diff
@@ -642,17 +642,15 @@ bool FT2Font::load_char_with_fallback(FT2Font *&ft_object_with_glyph,
                                       FT_Error &glyph_error,
                                       bool override = false)
 {
-    FT_UInt glyph_index = FT_Get_Char_Index(face, charcode);
-
+    FT_UInt glyph_index = ft_get_char_index_or_warn(face, charcode, false);
```

Comment:
I don't think this needs to go through the `_or_warn` path...

with `warn=false`, this is just equivalent to `FT_Get_Char_Index`, which was already being used... (if the return is non-zero it just returns it if it is zero, it checks `warn` and calls `ft_glyph_warn` if `warn` is true, then it returns zero)

Same goes for line 612 above

Reply:
I mean, I could see it being useful if warning was on some boolean flag, but passing `false` in directly seems not so useful. (Current main has one instance which doesn't pass warn at all, and one instance where it passes `false`, no instances of passing a variable... so if we don't use it like that, i'd be okay with removing the functionality)